docs/nidigital/class.rst

```diff
@@ -3384,6 +3384,18 @@ channels
         passes a string of :python:`'0, 1, 2'` to the set attribute function.
 
 
+pins
+----
+
+    .. py:attribute:: nidigital.Session.pins[]
+
+        .. code:: python
+
+            session.pins['0-2'].channel_enabled = True
+
+        passes a string of :python:`'0, 1, 2'` to the set attribute function.
```

Comment:
We need to stop documenting implementation details. The customer doesn't have to know or care how this ends up calling into the C API.

#1126 created